lib/elixir/test/elixir/kernel/errors_test.exs

```diff
@@ -648,6 +650,17 @@ defmodule Kernel.ErrorsTest do
     assert_compile_fail CompileError,
       "nofile:1: cannot use ^x outside of match clauses",
       'x = 8; <<a, b::size(^x)>> = <<?a, ?b>>'
+
+    warning = capture_io(:stderr, fn ->
```

Comment:
No, it is fine to leave it here, we shouldn't just not assert anything on the warning. @ggcampinho can you please make sure you don't assert a warning is shown? it is not the purpose of this test after all. :)